src/components/AddTimeEntry.tsx

```diff
@@ -0,0 +1,92 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { getFormattedDate } from "../utils/dateFunctions";
+import circledPlusIcon from '../assets/icons/circledPlusIcon.png'
+import { AddTimeEntryProp } from "../types/types";
+import { getSelectedProjectAndClient } from "../utils/getSelectedProjectAndClient";
+import { Project } from "./Project";
+
+export function AddTimeEntry(props: AddTimeEntryProp) {
+    return (
+        <>
+            <div className={props.isModalOpen ? "add-task-container" : "add-task-container zIndex"}>
+                <div className="description-project-container">
+                    <input
+                        type="text"
+                        placeholder="What are you working on?"
+                        className="input-box"
+                        onChange={props.onNameChange} 
+                        value={props.taskDescription}
+                        onKeyDown={props.onEnter}
+                    ></input>
+                    <button onClick={props.onToggle} className={!props.selectedProject.value ? "project-text-color project-text" : 'project-text'}>
+                        {!props.selectedProject.value && 
+                            <img 
+                                src={circledPlusIcon} 
+                                alt="Circled Plus Icon" 
+                                style={{ width: '20px', height: '20px', marginRight: '5px'}}
+                            />
+                        }
+                        {getSelectedProjectAndClient(props.selectedProject, props.selectedClient)}
+                    </button>
+                </div>
+                <div className="add-sub-container">
+                    <input
+                        className="startTimeBox"
```

Comment:
`<div className="description-project-container">` and here `startTimeBox` is in camel case. Stick to a single convention. Use either kebab case or camel case. Don't mix them. In general CSS classnames are denoted in kebab case but I leave that to you to set the standard here.

Reply:
I'll update the CSS class names to consistently use kebab case across the project, Anna.

Reply:
https://github.com/PreethiKalyani99/Clockify-Typescript/commit/a8de79fdbd489fa9e64c33deb480411aca3070a3